Source/WebCore/platform/graphics/avfoundation/objc/CDMSessionAVContentKeySession.mm

```diff
@@ -273,17 +279,20 @@ static bool isEqual(Uint8Array* data, const char* literal)
         m_initData = protectedSourceBuffer->initData();
     }
 
-    if (!m_keyRequest) {
+    if (!hasContentKeyRequest()) {
         RetainPtr<NSData> nsInitData = m_initData ? m_initData->createNSData() : nil;
         NSData* nsIdentifier = m_identifier ? [NSData dataWithBytes:m_identifier->data() length:m_identifier->length()] : nil;
         if ([contentKeySession() respondsToSelector:@selector(processContentKeyRequestWithIdentifier:initializationData:options:)])
             [contentKeySession() processContentKeyRequestWithIdentifier:nsIdentifier initializationData:nsInitData.get() options:nil];
         else
             [contentKeySession() processContentKeyRequestInitializationData:nsInitData.get() options:nil];
+        m_hasKeyRequestSemaphore.waitFor(5_s);
```

Comment:
A commend and a named constant would help someone looking at this understand why we wait for exactly 5 seconds.